docs/sla.md

```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
+
+
+## Definitions
+
+* "Self-service Console" provides access to a self-service console at https://console.cdnetworks.com. The console provides a way to configure the service through the Web UI or API. 
```

Comment:
We refer to "portal" in our documentation https://docs.cdnetworks.com/cdn/docs/portal/accessing-portal/logging-in .
Shouldn't we use that term instead?
The first sentence isn't helpful because you define the new phrase using the phrase itself. 
console.cdnetworks.com provides a web UI.   It allows the user to avoid using the API directly, so I'm not sure why you mention it in this context.

Reply:
Please consider changing the first sentence to: "The Self-service Console at https://console.cdnetworks.com  provides access to self-service features via a Web UI or API." Then we no longer need the second sentence.